string name
geometry_msgs/Pose pose
bool is_continuous
bool ignore_orientation
float64 intermediate_tolerance
bool ignore_obstacles
bool ignore_walls
string interruptable_by
